core/database/foxx/api/topic_router.js

```diff
@@ -71,14 +102,44 @@ router
 
 router
     .get("/view", function (req, res) {
+        const client = g_lib.getUserFromClientID(req.queryParams.client);
+        let topic = undefined;
         try {
+            logger.logRequestStarted({
+                client: client?._id,
+                correlationId: req.headers["x-correlation-id"],
+                httpVerb: "GET",
+                routePath: basePath + "/view",
+                status: "Started",
+                description: "View topic",
+            });
+
             if (!g_db.t.exists(req.queryParams.id))
                 throw [error.ERR_NOT_FOUND, "Topic, " + req.queryParams.id + ", not found"];
 
-            var topic = g_db.t.document(req.queryParams.id);
+            topic = g_db.t.document(req.queryParams.id);
 
             res.send([topic]);
+            logger.logRequestSuccess({
+                client: client?._id,
+                correlationId: req.headers["x-correlation-id"],
+                httpVerb: "GET",
+                routePath: basePath + "/view",
+                status: "Success",
+                description: "View topic",
+                extra: topic,
```

Comment:
This looks like it is returning the rev _key, and _id which is not really needed can we just return the title, creator, coll_cnt

```
           topic = obj.db.t.save(
                {
                    title: topic,
                    creator: a_owner_id,
                    coll_cnt: 1,
                },
                {
                    returnNew: true,
                },
            );
```